### PR TITLE
Replace tslint with eslint

### PR DESCRIPTION
**Pull request by joao-vitor-sr**
The tslint is obsolete since 2019 is good to replace with eslint, and the eslint has more features than tslint

### CODE REVIEW

File: test/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('Example class', () => {
 	});
 	it('should return whatever is passed to exampleMethod()', () => {
 		const example: Example = new Example();
-		const param: string = 'This is my param.';
```
Comment (chriswells0) [inline test/index.ts:13]: Why was the type removed here?

**Review by chriswells0 — CHANGES_REQUESTED** [review]
Thank you for making this change! I should have merged it first, but I saved it for last and now there's a merge conflict. Would you mind rebasing?

I also had a question about why the type was removed in a unit test.

File: test/index.ts (hunk 1 above)
Comment (joao-vitor-sr) [reply test/index.ts:13]: It is type inference the type is automatically set to string type

[Type Inference](https://www.typescriptlang.org/docs/handbook/type-inference.html)

**Comment by chriswells0** [comment]
Thanks again!

**Pull request merged**